app/screens/global_drafts/components/global_drafts_list/DraftSwipeActions/index.tsx

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useRef} from 'react';
```

Comment:
I notice the folder naming is different here than others.  DraftSwipeActions.  The pattern I'm seeing would have made this folder draft_swipe_actions.  I don't know if this is a rule/pattern but I think we ought to be consistent.

Reply:
Ah, will change it.